### ISSUE TITLE
[a11y] PartialDateUI change navigates to following selectable object

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1114]
**Describe the bug**
You cannot change the PartialDateUI format (Full Date, Month, Year, Year) by using arrow keys without switching back and forth between two selectable objects.

**To Reproduce**
1. Attempt to switch the PartialDateUI field with the arrow keys.

**Expected behavior**
Other picklists allow complete navigation of their options with the arrow keys alone.

**Comment by maxxxxxdlp** [comment]
Is this supposed to be two separate issues?

Also, I don't understand what do you mean about arrow keys and PartialDateUI. Up/down arrows are working for switching the date format (Full Date, Month, Year, Year). Then, press Tab and the date picker is focused.
Press Shift+Tab to focus date format selector again

https://user-images.githubusercontent.com/40512816/146216550-1faf757b-dfe9-4c57-8266-6b887ac90d67.mp4

**Comment by grantfitzsimmons** [comment]
I was expecting the keyboard navigation to behave as it does with any given pick list (changing the content in the pick list does not navigate to the next selectable object). You cannot currently go from Year to Full Date without navigating back and forth twice.

**Comment by grantfitzsimmons** [comment]
The title was an error! Apologies

**Comment by maxxxxxdlp** [comment]
Seems like the issue only happens on Windows

Works fine on macOS:
https://user-images.githubusercontent.com/40512816/146389893-c86b5dd9-72b9-4827-bdcf-755649d1b8d5.mp4

**Comment by maxxxxxdlp** [comment]
Fixed in d06de89c
